XMonad/Util/Font.hs

```diff
@@ -41,15 +41,15 @@ import Control.Exception as E
 import Text.Printf (printf)
 
 #ifdef XFT
-import Graphics.X11.Xft
 import Graphics.X11.Xrender
+import Graphics.X11.Xft
 #endif
 
 -- Hide the Core Font/Xft switching here
 data XMonadFont = Core FontStruct
                 | Utf8 FontSet
 #ifdef XFT
-                | Xft  XftFont
+                | Xft  (NonEmpty XftFont)
```

Comment:
This is debatable, one might also argue that `[XftFont]` fits better with the rest of the module and functions; use your own judgement here